R/pipeline_tumor_normal.R

```diff
@@ -0,0 +1,87 @@
+
+#' Call CAMDAC for a tumor and patient-matched normal sample
+#'
+#' @param patient_id character. Patient identifier
+#' @param tumor_id character. Tumor sample identifier
+#' @param normal_id character. Normal sample identifier
+#' @param tumor_bam character. Full path to tumor bam file
+#' @param normal_bam character. Full path to normal bam file
+#' @param sex character. Patient sex: "XX" for female or "XY" for male
+#' @param path character. Full path to CAMDAC output directory
+#' @param pipeline_files character. Full path to parent directory containing CAMDAC pipeline_files
+#' @param build character. Genome build: "hg19" or "hg38"
+#' @param min_tumor integer. Minimum read filter for tumor samples
+#' @param min_normal integer. Minimum read filter for normal samples
+#' @param n_cores integer. Number of cores to use for parallel processing
+#' @param mq integer. Minimum mapping quality filter
+#' @export
+pipeline_tumor_normal <- function(patient_id, tumor_id, normal_id, tumor_bam, normal_bam, sex, path,
```

Comment:
Check if repeat elements are coloured in DMR plots

Reply:
As discussed, I wonder if they are even included in the DMR object at all. If not, perhaps this is better left for the next iteration of updates. Should we create a tracker of next steps?